src/DrawTogether.Actors/Local/LocalPaintProtocol.cs

```diff
@@ -28,25 +29,17 @@ public sealed class LeavePaintSession(DrawingSessionId drawingSessionId, UserId
 
     public sealed class AddPointToConnectedStroke(
         Point point,
-        StrokeId strokeId,
         DrawingSessionId drawingSessionId,
-        UserId userId)
+        UserId userId, GreaterThanZeroInteger strokeWidth, Color strokeColor)
         : IPaintSessionMessage
     {
         public Point Point { get; } = point;
-
-        public StrokeId StrokeId { get; } = strokeId;
-        public DrawingSessionId DrawingSessionId { get; } = drawingSessionId;
-        public UserId UserId { get; } = userId;
-    }
-
-    public sealed class CreateConnectedStroke(DrawingSessionId drawingSessionId, UserId userId, ConnectedStroke connectedStroke)
```

Comment:
Don't have the Blazor UI decide how or when to group strokes - have the debouncer do it.